src/formGenerator.vue

```diff
@@ -1,6 +1,7 @@
 <template lang="pug">
 div
 	fieldset.vue-form-generator(v-if='schema != null', :is='tag')
+		legend(v-if='schema.legend') {{ schema.legend }}
```

Comment:
I would add a bit more of logic in here to not need more than one tag <vue-form-generator></vue-form-generator>